.orchestra/config/components/expat.yml

```diff
@@ -0,0 +1,23 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+#@ load("/lib/cmake.lib.yml", "typical_cmake_builds")
+
+---
+#@ def build_args():
+test: false
+build_dependencies:
+  - cmake
+  - glibc
+  - host-cxx-toolchain
+use_asan: false
+source_url: "https://github.com/libexpat/libexpat/releases/download/R_2_4_8/expat-2.4.8.tar.gz"
+#@ end
+
+#@overlay/match by=overlay.all, expects=1
+#@overlay/match-child-defaults missing_ok=True
+---
+components:
+  expat:
+    license: COPYING
+    default_build: optimized
+    builds: #@ typical_cmake_builds(**build_args())
```

Comment:
This will create three builds, we don't want this for components that we expect not to change.
However apparently we have a precedent, `libfort`, so I think we can keep it as it is.